docs/developer/other/API.md

```diff
@@ -106,3 +106,47 @@ Example response:
   }
 }
 ```
+### Portlet Preferences
+
+`/api/prefs/{task}`
+
+(example: `http://localhost:8080/uPortal/api/prefs/getprefs/weather` )
```

Comment:
At first glance, it seems we should support `GET` (read) and `PUT` (update) out of the gate.

I'm not sure if it makes sense to support `POST` (create), because a basic collection of favorites is created whenever you publish in the first place.

Reply:
@drewwills I am already supporting gets and puts